governance/votes/Executive vote - December 3, 2021.md

```diff
@@ -0,0 +1,136 @@
+
+---
+
+## Proposal Details
+
+### MOMC Parameter Changes
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmNqCZGa?network=mainnet#poll-detail), the following parameter changes will be made in the Maker Protocol if this executive proposal passes.
+- Increase the ETH-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 2.5% to 2.75%.
+- Increase the ETH-B [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 6.0% to 6.5%.
+- Increase the LINK-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 1.5% to 2.5%.
+- Increase the MANA-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 3.0% to 6.0%.
+- Increase the UNI-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 1.0% to 3.0%.
+- Increase the GUSD-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 0.0% to 1.0%.
+- Increase the UNIV2DAIETH-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 1.5% to 2.0%.
+- Increase the UNIV2WBTCETH-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 2.5% to 3.0%.
+- Increase the UNIV2USDCETH-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 2.0% to 2.5%.
+- Increase the UNIV2UNIETH-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 2.0% to 4.0%.
+- Decrease the GUNIV3DAIUSDC1-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 0.5% to 0.1%.
+- Increase the WBTC-A [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 1.5 billion DAI to 2 billion DAI.
+- Increase the WBTC-A [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam) from 60 million DAI to 80 million DAI.
+
+### [Debt Floor (`dust`)](https://makerdao.world/en/learn/governance/param-debt-floor) Increases
+
+As per these successful governance polls, relating to [ETH-B](https://vote.makerdao.com/polling/QmZXnn16?network=mainnet) and other [vaults (excluding ETH-B and ETH-C)](https://vote.makerdao.com/polling/QmUYLPcr?network=mainnet), the following [Debt Floor (dust)](https://makerdao.world/en/learn/governance/param-debt-floor) changes will be made to the Maker Protocol if this executive proposal passes.
+- Increase the ETH-B [Debt Floor (`dust`)](https://makerdao.world/en/learn/governance/param-debt-floor) from 30,000 DAI to 40,000 DAI.
+- Increase the [Debt Floor (`dust`)](https://makerdao.world/en/learn/governance/param-debt-floor) from 10,000 DAI to 15,000 DAI on all eligible vaults, **excluding**:
+	- ETH-B
+	- ETH-C
+	- Direct Deposit Module Vaults
+	- Peg Stability Module Vaults
+	- Real World Asset Vaults
+
```

Comment:
Would add link to the forum post for further details for this one definitely

Reply:
Yep